style.css

```diff
@@ -0,0 +1,16 @@
+body {
+  background-color: darkolivegreen;
+}
+
+h1 {
+  text-align: center;
+  color: aliceblue;
+}
```

Comment:
Here, instead of using HTML tags (body and h1), why don't we use classes instead to select specific elements in our HTML to style? It will make a great difference in the future and we will avoid a lot of bugs like styles applied to elements that we don't want to, what do you think about this suggested change?